packages/relay/tests/lib/eth/eth_getBlockByNumber.spec.ts

```diff
@@ -399,30 +412,30 @@ describe('@ethGetBlockByNumber using MirrorNode', async function () {
     it('eth_getBlockByNumber with finalized tag', async function () {
       const result = await ethImpl.getBlockByNumber('finalized', false);
       // check that we only made the expected number of requests with the expected urls
-      expect(restMock.history.get.length).equal(4);
+      expect(restMock.history.get.length).equal(12);
```

Comment:
Where is this number coming from, can't we get it dynamically instead of hardcoding it? Same for other occurrences below.

Reply:
Good point, if we can't get it dynamically, we should set them in a variable, instead of using magic number, for better readabillity.

Reply:
I don't think we can get it dynamically, so I added it to a constant. In general, this can also be refactored and the test can be made more specific, but I think it is out of this PR's scope